main.go

```diff
@@ -1,11 +1,35 @@
 package main
 
 import (
-	"net/http"
-	"url-shortener/routes"
+	"log"
+	"os"
+
+	"tiny-site-backend/initializers"
+	"tiny-site-backend/routes"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/cors"
+	"github.com/gofiber/fiber/v2/middleware/logger"
 )
 
+func init() {
+	config, err := initializers.LoadConfig(".")
+	if err != nil {
+		log.Fatalln("Failed to load environment variables! \n", err.Error())
```

Comment:
log.Fatal() also calls `os.Exit()` internally. So do we need this?

Reply:
no I have changed to log.Printf